docs/sources/operators-guide/monitor-grafana-mimir/collecting-metrics-and-logs.md

```diff
@@ -112,30 +112,16 @@ metaMonitoring:
 You can also send the collected metamonitoring metrics to the installation of Mimir or GEM.
 The configuration varies slightly for GEM and Mimir.
 
-If you have deployed Mimir, then the Helm chart values should look like the following example.
-Replace `HELM_RELEASE_NAME` with the name of your Helm release:
-
-```yaml
-metaMonitoring:
-  serviceMonitor:
-    enabled: true
-  grafanaAgent:
-    enabled: true
-    installOperator: true
-
-  metrics:
-    remote:
-      url: "http://<HELM_RELEASE_NAME>-mimir-nginx.mimir.svc/api/v1/push"
-      headers:
-        X-Scope-OrgID: metamonitoring
-```
-
-If you have deployed GEM, then the URL will point to the GEM gateway instead of the nginx Deployment.
-If you are using the GEM authentication model, then you also need to provide a Secret with the
-authentication token for a tenant. Refer to [Credentials](#credentials) for setting up the Secret.
+If you have deployed Mimir, and the URL is not set, then metrics will be sent to the Mimir cluster.
+You can query these metrics using the `__metamonitoring__` tenant.
 
+If you have deployed GEM, then you need to configure the remote endpoint. The URL should point to
```

Comment:
The calculation works for GEM as well, so GATEWAY_URL need not be replaced, just leave it empty, right?

Reply:
won't work with GEM auth. I can also use the config and see if `auth.type == enterprise`, but that was too complex to explain, so I figured it's a complicated behaviour and decided not to do it

Reply:
I think you have to say something about authentication and access policies/tokens in any case. So I'd rather not make the user copy the URL if we can calculate it easily.

Reply:
i say something about them. but my point is that now i need to explain that if they set auth.type:enterprise anywhere in their values files or they are using the defaults from the chart, theeen they need to setup some access policy. 

this is too long to explain and the more complex it is the more people will find it confusing

i will push a commit for you to see

Reply:
I think saying something about access policies is independent of whether we default the URL or not.
As soon as we start to document sending metrics back to Mimir/GEM, it comes up.

The only way to not talk about this is to auto-provision some policy in case auth is enterprise.